src/main/java/mate/academy/service/UserService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.service;
+
+import java.util.Optional;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.User;
+
+public interface UserService {
+    User add(User user);
+
+    Optional<User> findByEmail(String email) throws AuthenticationException;
```

Comment:
according the task you do not need to use throws AuthenticationException in method findByEmail

Reply:
++